xbmc/interfaces/python/PythonSwig.cpp.template

```diff
@@ -117,6 +126,8 @@ void doMethod(Node method, MethodType methodType)
     else
       return;
   }
+  else if (method.@name == "contains")
```

Comment:
Yeah because I ignore `contains` (and `toString`) in the regular list of methods handled by `doMethod()`, see https://github.com/xbmc/xbmc/pull/6264/files#diff-899036c0c39c2c424042c8102edc523bR379. I checked and there's no `contains` method in the current legacy API.
Do you have a better idea how to decide (maybe more explicit) whether an instance of a class can be used as a sequence?